### DISCUSSION TITLE
Easy (Default) Shortcuts

### DISCUSSION THREAD

**Discussion by maymage** [flxzt/rnote#732] [Ideas]
The operational shortcuts, those for really working on the document, are just a few 

* `Ctrl+{1,...,6}` -> select shapes
* `Ctrl+z` -> undo
* `Ctrl+c`
*  `Ctrl+v`

Those are not all ergnonomically not really nice, when one hand is holding the pen, 
as you're forced to press two buttons with one hand at the same time. 

Suggestion: single letter shortcuts, 

* `{1,..,6}` to select the shapes -> this can be easily remembered and seen from the UI
* `<-` and `->` for undo and redo.

Caveat: `3` would lead us to the text entry, where we have to be able to use the keyboard differently. 
We would then need to press `Escape` to return to the ground state. 

---

Other possible shortcuts

* `a` -> arrows
* c -> circle **as opposed to `Ctrl+c` to copy**
* r -> rectangle
* etc... 

that is, shortcuts with mnemonics.

**Comment by flxzt** [comment]
A bit late, but I am sceptical about adding shortcuts that are not a combination of `<ctrl>+...` because it might interfere with the app in unexpected ways/is difficult to implement in combination with the typewriter
